website/static/js/registrationRetraction.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * On page load, focuses on justification input and
```

Comment:
It's not anymore. I had originally been using jQuery to do this but have since moved to HTML5's autofocus attribute